runtime/src/lib.rs

```diff
@@ -1426,6 +1426,21 @@ impl_runtime_apis! {
             result.encode()
         }
 
+        fn get_subnet_info_v2(netuid: u16) -> Vec<u8> {
+            let _result = SubtensorModule::get_subnet_info_v2(netuid);
+            if _result.is_some() {
+                let result = _result.expect("Could not get SubnetInfo");
+                result.encode()
+            } else {
+                vec![]
+            }
+        }
```

Comment:
Runtime apis can return Options / Results

Reply:
With exepction of deprecated fields , this is a like for like of SubentInfo. We can apply any optimisations / nits in a separate  pr. this is to allow subnet identities go out without breaking changes